<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDeviceDialog</class>
 <widget class="QDialog" name="AddDeviceDialog">
  <property name="geometry"><rect><x>0</x><y>0</y><width>500</width><height>350</height></rect></property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0"><widget class="QLabel" name="lbl_name"><property name="text"><string>Device Name:</string></property></widget></item>
   <item row="0" column="1"><widget class="QLineEdit" name="le_name"/></item>

   <item row="1" column="0"><widget class="QLabel" name="lbl_type"><property name="text"><string>Device Type:</string></property></widget></item>
   <item row="1" column="1"><widget class="QComboBox" name="cb_type"/></item>

   <item row="2" column="0"><widget class="QLabel" name="lbl_energy"><property name="text"><string>Energy Usage:</string></property></widget></item>
   <item row="2" column="1"><widget class="QLineEdit" name="le_energy"/></item>

   <item row="3" column="0"><widget class="QLabel" name="lbl_status"><property name="text"><string>Status:</string></property></widget></item>
   <item row="3" column="1"><widget class="QComboBox" name="cb_status">
    <item><property name="text"><string>ON</string></property></item>
    <item><property name="text"><string>OFF</string></property></item>
   </widget></item>

   <item row="4" column="0"><widget class="QLabel" name="lbl_malfunction"><property name="text"><string>Malfunction:</string></property></widget></item>
   <item row="4" column="1"><widget class="QComboBox" name="cb_malfunction">
    <item><property name="text"><string>False</string></property></item>
    <item><property name="text"><string>True</string></property></item>
   </widget></item>

   <item row="5" column="0"><widget class="QLabel" name="lbl_district"><property name="text"><string>Assign to District:</string></property></widget></item>
   <item row="5" column="1"><widget class="QComboBox" name="cb_district"/></item>

   <item row="6" column="1"><widget class="QPushButton" name="btn_add_device"><property name="text"><string>Add Device</string></property></widget></item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
